cmd/starter/c/starter.c

```diff
@@ -316,10 +316,12 @@ static void apply_privileges(struct privileges *privileges, struct capabilities
                 fatalf("Failed to set GID %d: %s\n", targetGID, strerror(errno));
             }
 
-            debugf("Set %d additional group IDs\n", privileges->numGID);
-            if ( setgroups(privileges->numGID, privileges->targetGID) < 0 ) {
-                fatalf("Failed to set additional groups: %s\n", strerror(errno));
-            }
+	    if ( privileges->numGID > 1 ) {
+		debugf("Set %d additional group IDs\n", privileges->numGID);
```

Comment:
bad indentation

Reply:
Good catch.  I had vim set up to not expand tabs, for go.  I changed it to expand tabs to spaces for `*.[ch]` files.